src/frontend/Komponenter/Behandling/Simulering/SimuleringTabellWrapper.tsx

```diff
@@ -71,6 +76,13 @@ const SimuleringTabellWrapper: React.FC<{
     return (
         <SimuleringsContainer>
             <SimuleringOversikt simulering={simuleringsresultat} />
+            {simuleringsresultat.sumManuellePosteringer != null &&
+                simuleringsresultat.sumManuellePosteringer == 0 && (
```

Comment:
Er det ikke motsatt - hvis  `sumManuellePosteringer !==0` ?

Reply:
Ja, det var bare for å teste / se warningen